### PR TITLE
DOMA-2656 Use spec char validator for employee name

### PR DESCRIPTION
**Pull request by VKislov**
Added a special character validator for the employee name on the create page. Some refactoring and fix for a potential crash with `TypeError: Cannot read property of undefined` due to using `link: { role }`

### CODE REVIEW

**Review by sitozzz — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

File: apps/condo/pages/employee/index.tsx [hunk 1]
```diff
@@ -203,8 +203,9 @@ const EmployeesPage = () => {
     const intl = useIntl()
     const translations = intl.messages
 
-    const { organization, link: { role } }  = useOrganization()
-    const userOrganizationId = get(organization, 'id', null)
+    const { link }  = useOrganization()
+    const role = get(link, 'role', {})
+    const userOrganizationId = get(link, ['organization', 'id'], {})
```
Comment (akarjakin) [inline apps/condo/pages/employee/index.tsx:208]: did `useOrganization` change and `organization` move to `link` ?

[the remark this answers is not in the document]
Comment (VKislov) [reply apps/condo/pages/employee/index.tsx:208]: `organization` and `link.organization` are the same
packages/next/organization.jsx:133

File: apps/condo/pages/property/index.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /** @jsx jsx */
+import get from 'lodash/get'
```
Comment (AleX83Xpert) [inline apps/condo/pages/property/index.tsx:2]: Is this import still necessary?

File: apps/condo/pages/meter/create.tsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import Head from 'next/head'
 import React from 'react'
+import get from 'lodash/get'
```
Comment (AleX83Xpert) [inline apps/condo/pages/meter/create.tsx:3]: Unused import?

**Review by AleX83Xpert — APPROVED** [review]

**Pull request merged**